### PR TITLE
Adding logging overview

### PR DESCRIPTION
**Pull request by CatGarab**

### CODE REVIEW

**Comment by CatGarab** [comment]
[update documentation for how logging works and is accessed](https://trello.com/c/hpu5jH1H/154-update-documentation-for-how-logging-works-and-is-accessed)

File: docs/os-docs/logging.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+Within each of these sub-directories, the following files may be automatically created:
+
+- ``debug.log`` - Records all log messages
+- ``info.log`` - Records log messages with a priority of ``info`` or higher
+- ``warn.log`` - Records log messages with a priority of ``warn`` or higher
+
+Log Rotation
+~~~~~~~~~~~~
+
+All log files have a maximum size of 1MB.
```
Comment (jacoffey3) [inline docs/os-docs/logging.rst]: "of 1MB by default."

Comment (jacoffey3) [reply docs/os-docs/logging.rst]: Please put a note here pointing them to where they would change that

File: docs/os-docs/logging.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+Once this size is reached, the current file is renamed as an archive file and a new log file is
+started. Archive files use their original name, but are suffixed with the current timestamp.
+For example, ``debug.log.2018.12.01-00.12.07``.
+
+By default, five archive files of each log type will be retained.
```
Comment (jacoffey3) [inline docs/os-docs/logging.rst]: Add a note saying where to change this as well (or rather put them in the same note)

**Pull request merged**